fdbserver/WorkerInterface.actor.h

```diff
@@ -149,9 +149,9 @@ struct ClusterControllerFullInterface {
 	RequestStream<struct RegisterWorkerRequest> registerWorker;
 	RequestStream<struct GetWorkersRequest> getWorkers;
 	RequestStream<struct RegisterMasterRequest> registerMaster;
-	RequestStream<struct UpdateWorkerHealthRequest> updateWorkerHealth;
 	RequestStream<struct GetServerDBInfoRequest>
 	    getServerDBInfo; // only used by testers; the cluster controller will send the serverDBInfo to workers
+	RequestStream<struct UpdateWorkerHealthRequest> updateWorkerHealth;
```

Comment:
Just a FYI: This change (as well as the one in `operator==`) is unnecessary. The only one you need for protocol compatibility is the one in `serialize`

Reply:
Thanks for the info! and it makes sense.